com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -913,8 +912,15 @@ public void Shutdown()
                 BehaviourUpdater = null;
             }
 
-            //The Transport is set during Init time, thus it is possible for the Transport to be null
-            NetworkConfig?.NetworkTransport?.Shutdown();
+            // This is required for handling the potential scenario where multiple NetworkManager instances are created.
+            // See MTT-860 for more information
+            if (IsListening)
+            {
+                //The Transport is set during initialization, thus it is possible for the Transport to be null
+                NetworkConfig?.NetworkTransport?.Shutdown();
```

Comment:
there's a bigger question here which is: how we associate multiple NetworkManager instances with transports? For UNET, looks like we have to use the same transport instance (singleton) but that might not be the case for UTP, every NetworkManager could create its own UTP NetworkDriver etc. just as an FYI, I'd like to summon @andrews here :)

Reply:
So in the case of uNET that is correct we only have a singleton for the NetworkTransport object, in the case of UTP we would do exactly that and just create a new driver per instance of the transport.

Reply:
Yeah, that would be great if we could but we still would need to check to make sure multiple NetworkManager instances was intended or not.  If we get multiprocess unit tests to replace multiInstance, then the next change to NetworkSceneManager (i.e. SceneManagerAPI) would provide us with a way to create 1 NetworkManager.  We could have different configurations that the NetworkManager can "load" (i.e. ScriptableObjects), and then NetworkManager sort of becomes a "singleton" again.